src/Build.UnitTests/BackEnd/SdkResolverService_Tests.cs

```diff
@@ -28,20 +29,24 @@ public class SdkResolverService_Tests : IDisposable
     {
         private readonly MockLogger _logger;
         private readonly LoggingContext _loggingContext;
+        private readonly EventSourceTestHelper _eventSourceTestListener;
 
         public SdkResolverService_Tests()
         {
             _logger = new MockLogger();
             ILoggingService loggingService = LoggingService.CreateLoggingService(LoggerMode.Synchronous, 1);
             loggingService.RegisterLogger(_logger);
 
+            _eventSourceTestListener = new EventSourceTestHelper();
```

Comment:
_eventSourceTestListener is used only in one test but is it initialized for every test. Since listening to event sources have performance impact, all test which do not test events will have to pay this performance. 
Please consider to not initialize it in constructor and use `using EventSourceTestHelper eventSourceTestHelper = new()` in particular test(s) instead.